Sources/TuistLoaderTesting/Utils/Mocks/MockManifestFilesLocator.swift

```diff
@@ -8,7 +8,11 @@ public final class MockManifestFilesLocator: ManifestFilesLocating {
 
     public func locate(at: AbsolutePath) -> [(Manifest, AbsolutePath)] {
         locateArgs.append(at)
-        if let locateStub = locateStub { return locateStub }
-        return [(.project, at.appending(component: "Project.swift"))]
+        return locateStub ?? [(.project, at.appending(component: "Project.swift"))]
+    }
+    
+    public func locateAll(at: AbsolutePath) -> [(Manifest, AbsolutePath)] {
+        locateArgs.append(at)
```

Comment:
We should not reuse `locateArgs` because that attribute is associated to the `locate` method. We should define `locateAllArgs` and `locateAllStub`.